source/instances/object_concat.js

```diff
@@ -0,0 +1,10 @@
+export default function object_concat(a, b) {
+  var c = {};
+  for (var k in a) {
+    c[k] = a[k];
+  }
+  for (var k in b) {
+    c[k] = b[k];
+  }
+  return c;
+};
```

Comment:
This is semantically different from the Sanctuary implementation. See sanctuary-js/sanctuary-type-classes#70.

Reply:
Probably should use internal `keys`